docs/8.4-defaults-and-tuning.md

```diff
@@ -0,0 +1,56 @@
+# Defaults and tuning guidance for {{vers}}
+
+MySQL {{vers}} updates several server defaults to align with modern CPUs, memory sizes, and SSD/NVMe storage. An in-place upgrade that blindly reuses an 8.0-era `my.cnf` may forgo these improvements or cause unexpected performance behaviors. Review and re-evaluate configuration on {{vers}}—or generate a new config—rather than carrying old settings forward.
+
+## Notable InnoDB default changes
+
+| InnoDB System Variable Name | New Default ({{vers}}) | Previous Default (8.0) |
+| --- | --- | --- |
+| `innodb_adaptive_hash_index` | OFF | ON |
+| `innodb_change_buffering` | none | all |
+| `innodb_doublewrite_files` | 2 | `innodb_buffer_pool_instances * 2` |
+| `innodb_flush_method` on Linux | O_DIRECT if supported, otherwise fsync | fsync |
```

Comment:
`innodb_doublewrite_pages` changed from 4 to 128
https://perconadev.atlassian.net/browse/PS-10236